mep/common/admin.py

```diff
@@ -80,6 +80,26 @@ def before_import_row(self, row, **kwargs):
         """
         pass
 
+    def validate_row_by_slug(self, row):
+        if not row.get("slug"):
+            return False
+        if not self.Meta.model.objects.filter(slug=row["slug"]).exists():
+            # past slug?
+            work = self.Meta.model.objects.filter(past_slugs__slug=row["slug"]).first()
+            if work:
+                logger.debug(f'renaming {row["slug"]} to {work.slug}')
```

Comment:
your variable here is `work` but it looks like you've implemented for both person and works? (which btw is great); you could call it `instance` or `record`

I don't think this log message out of context would make sense to me; here's a suggested revision

```suggestion
                logger.debug(f'Record found by past slug {row["slug"]}, updating to {work.slug}')
```